src/main/webapp/app/pages/AccountPage.tsx

```diff
@@ -134,6 +139,54 @@ export class AccountPage extends React.Component<IRegisterProps> {
     if (this.account === undefined) {
       return <Redirect to={PAGE_ROUTE.LOGIN} />;
     }
+
+    const apiAccess = (
+      <InfoRow
+        title={
+          <div className={'d-flex align-items-center'}>
+            <span>
+              {getAccountInfoTitle(
+                ACCOUNT_TITLES.API_TOKEN,
+                this.account.licenseType as LicenseType
+              )}
+            </span>
+            {this.generateTokenEnabled ? (
+              <DefaultTooltip placement={'top'} overlay={'Get a new token.'}>
+                <i
+                  className={classnames('ml-2 fa fa-plus')}
+                  onClick={this.addNewToken}
+                />
+              </DefaultTooltip>
+            ) : null}
+          </div>
+        }
+      >
+        <TokenInputGroups
+          changeTokenExpirationDate={false}
+          tokens={this.tokens}
+          onDeleteToken={this.deleteToken}
+        />
+      </InfoRow>
+    );
+    const noApiAccess = (
+      <span>
+        You do not have API access. Click{' '}
+        <span
+          style={{ cursor: 'pointer' }}
+          onClick={() => (this.showConfirmModal = true)}
+        >
+          HERE
+        </span>{' '}
+        to request API access.
+      </span>
+    );
+    const awaitingApiAccess = (
+      <span>
+        We are reviewing your API access request. If you haven't heard back from
+        us after 5 business days, please feel free to contact us.
```

Comment:
Wondering why we say 5 business days to approve api access? Currently for account approval, we say 1-2 business days.